pose_format/utils/openpose.py

```diff
@@ -213,28 +214,41 @@ def load_openpose(frames: OpenPoseFrames, fps: float = 24, width: int = 1000, he
     return Pose(header, body)
 
 
-def get_frame_id(filename: str) -> int:
+def get_frame_id(filename: str, pattern: str) -> int:
     """
     Parses a filename to find the ID of a frame. Example file name for frame with ID 17:
     `CAM2_000000000017_keypoints.json`.
 
     :param filename: Name of openpose frame file.
     :return: Frame ID as an integer.
     """
-    pattern = "(?:^|\D)(\d+)\\_keypoints.json"
     m = re.findall(pattern, filename)
     frame_id = int(m[-1])
 
     return frame_id
 
 
-def load_openpose_directory(directory: str, fps: float = 24, width: int = 1000, height: int = 1000,
-                            depth: int = 0, num_frames: Optional[int] = None) -> Pose:
+def load_frames_directory_dict(directory: str, pattern: str) -> OpenPoseFrames:
```

Comment:
Why is this method called "dict" at the end? And would it perhaps make sense to move elsewhere if this is for both mediapipe and openpose?

Reply:
It is loading a dictionary. I agree that the placement is not optimal, but I am not pushing the surrey loading code to the repo, so it wouldn't make much sense elsewhere

Reply:
Oh sorry I missed that `load_openpose_directory` is calling this function called "...dict". All good!